### PR TITLE
Onboard eclipse-che org in openshift ci

### PR DESCRIPTION
**Pull request by flacatus**
Run tests against all branches wich start with 7.* related with releases and run against master branch.

Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### CODE REVIEW

**Comment by flacatus** [comment]
all jobs `ci/rehearse/eclipse-che/che-operator/7/*` are expected to fail

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash

**Review by alvaroaleman — APPROVED** [review]

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -3454,7 +3465,11 @@ lgtm:
   - openshift/ceph-csi
   review_acts_as_lgtm: true
 - repos:
+<<<<<<< HEAD
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_plugins.yaml:3468]: I don't think this will pass tests

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**